src/storage/heap_file.c

```diff
@@ -20602,6 +20613,21 @@ heap_delete_bigone (THREAD_ENTRY * thread_p, HEAP_OPERATION_CONTEXT * context, b
       log_addr.pgptr = context->overflow_page_watcher_p->pgptr;
       log_addr.vfid = &context->hfid.vfid;
       log_addr.offset = overflow_oid.slotid;
+      if (is_supplement)
+	{
+	  RECDES ovf_recdes = RECDES_INITIALIZER;
+	  if ((rc =
+	       heap_get_bigone_content (thread_p, context->scan_cache_p, COPY, &overflow_oid,
```

Comment:
```suggestion
	       heap_get_bigone_content (thread_p, context->scan_cache_p, PEEK, &overflow_oid,
```
it seems that `PEEK` is enough, and this is much faster.

Reply:
So do other `heap_get_*()`s.

Reply:
Thanks, I will change it.